Anchal_Hora/nodejs/Login.js

```diff
@@ -0,0 +1,34 @@
+function func(){
+    window.location.href = "./admin/index.html";
+    }
+document.getElementById("button").addEventListener("click",Gettoken)
+function Gettoken(){
+
+    var Sno=parseInt(document.getElementById("inputid").value);
+    var Username=document.getElementById("inputName").value;
+    var Email=document.getElementById("inputEmail").value;
+    var Password=document.getElementById("inputPassword").value;
+    
+    
+    var get = {
```

Comment:
`get` is not the right name for the request object

Reply:
Changed the name into json